cdap-ui/app/cdap/components/FieldLevelLineage/v2/Table/index.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+import React from 'react';
+import withStyles, { WithStyles } from '@material-ui/core/styles/withStyles';
+import { createStyles } from '@material-ui/core';
```

Comment:
This will include the entire material-ui/core module. We should see if this is necessary and if so this should be imported from a submodule. This will help minimizing the libraries we need.

Reply:
I've fixed this so we are no longer importing the entire material-ui/core module.